src/internal_events/aws_ecs_metrics.rs

```diff
@@ -13,13 +13,17 @@ pub struct AwsEcsMetricsReceived {
 
 impl InternalEvent for AwsEcsMetricsReceived {
```

Comment:
Per [the Events spec](https://github.com/vectordotdev/vector/blob/master/docs/specs/component.md#events), internal events indicating events received should be named in the format of `<Component>Event(s?)Received`.

In this case, `AwsEcsMetricsEventsReceived` seems accurate.